records/config.go

```diff
@@ -165,6 +173,14 @@ func SetConfig(cjson string) Config {
 		c.caPool = pool
 	}
 
+	if c.CertFile != "" {
```

Comment:
it's probably an error if one of these new config params is set, but not the other. any value in having a smarter check here for sanity?